INSTALL.md

```diff
@@ -125,6 +114,36 @@ Make sure the value of `--docker-server` is a valid [URI authority](https://data
   - `--docker-username` should be `_json_key`;
   - `--docker-password` should be the JSON-formatted access token for a service account that has permission to manage images in GCR.
 
+## Install Korifi
+
+Korifi is distributed as a [Helm chart](https://helm.sh). See [_Customizing the Chart Before Installing_](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing) for details on how to specify values when installing a Helm chart.
+
+For example:
+
+```sh
+helm install korifi https://github.com/cloudfoundry/korifi/releases/download/v<VERSION>/korifi-<VERSION>.tgz \
+    --namespace="$KORIFI_NAMESPACE" \
```

Comment:
Do we also need to pass `--create-namespace` so that helm creates the korifi namespace?

Reply:
people _could_ do that if they want, but I don't think it should be in the docs - we ask for a namespace, it's up to them how they get one